contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -743,6 +743,13 @@ Incorporating the default values into the `Spec` ensures that `Spec` depicts the
 full desired state so that it is easier for the system to determine how to
 achieve the state, and for the user to know what to anticipate.
 
+Default values can be specified on a field using the `+default=` tag. Primitives
+will have their values directly assigned while structs will go through the
```

Comment:
I thought we never converged on the struct default via JSON thing, did we already merge that?

Reply:
https://github.com/kubernetes/kube-openapi/blob/master/test/integration/testdata/defaults/default.go#L11-L12

Reply:
ok, great.